core/internal/featurechecker/features_test.go

```diff
@@ -0,0 +1,99 @@
+package featurechecker_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wandb/wandb/core/internal/featurechecker"
+	"github.com/wandb/wandb/core/internal/gqlmock"
+	spb "github.com/wandb/wandb/core/pkg/service_go_proto"
+)
+
+func makeServerFeatures() (*featurechecker.ServerFeatures, *gqlmock.MockClient) {
+	mockGQL := gqlmock.NewMockClient()
+	mockGQL.StubMatchOnce(
+		gqlmock.WithOpName("ServerFeaturesQuery"),
+		`{
```

Comment:
minor suggestion, to move this const out and just have the setup inside each test, this way 
it will be less awkward with the return values.

Reply:
Sounds like a good suggestion! I will only keep the stubbing for the GQL query to avoid needing to type that every time we want to add a new test.